docs/src/operations/requirements.md

```diff
@@ -48,7 +48,7 @@ releases at [solanalabs/solana](https://hub.docker.com/r/solanalabs/solana).
 
 ## Software
 
-- We build and run on Ubuntu 20.04.
+- We build and run on Ubuntu 24.04.
```

Comment:
While I appreciate the sentiment here with Ubuntu 20.04 going EOL this month, there are more things that need to happen on the backside to make this happen. For example, the release bins are compiled on an Ubuntu 20.04 machine for sake of backwards compat. So, we can't update this comment until we update that.

This is something that is on our radar, and changes likely coming soon

Reply:
Ha, I now see Trent's post in Discord and his PR https://github.com/anza-xyz/agave/pull/6067; think we'll probably go with that. Appreciate the hustle tho !